source/production/F0.Minesweeper.Components/Difficulty.razor

```diff
@@ -0,0 +1,9 @@
+<div>
+	<label for="difficulty">Difficulty</label>
+	<select @bind="Selected" name="difficulty">
+		@foreach (var value in Enum.GetNames(typeof(F0.Minesweeper.Components.Abstractions.Enums.DifficultyLevel)))
```

Comment:
We could either put the using for `F0.Minesweeper.Components.Abstractions.Enums` at the top of the razor file or add it to the `_Imports.razor` of the component library.

Reply:
The designer gave me a hard time. I found out later, that I should have closed the file and reopen it. But I have not thought of the _Imports. razor file. Sounds good